dependencies/build_libheif.sh

```diff
@@ -20,6 +20,8 @@ cd ..
 
 mkdir -p /output/lib /output/include /output/pkgconfig
 cp -a /usr/local/lib/libheif* /output/lib/
+cp -a /usr/lib/${DEB_HOST_MULTIARCH}/libheif/plugins/libheif-*.so /output/lib
+cp -a /usr/local/lib/${DEB_HOST_MULTIARCH}/gdk-pixbuf-*/*/loaders/libpixbufloader-heif.so /output/lib
```

Comment:
So, it looks like the plugins are installed during the build, but we forgot to copy them to the final image.
Is there a chance, anything else is forgotten?

BTW, I see that these `cp` commands don't save the original location paths of copied *.so files. Are you sure that it is fine and the lib will find them on the final image even if they are located not on the same paths as were installed during the build?

Reply:
There is no other files for `libraw`. For `imagemagick`, I see some configuration xml which should be included, like `quantization-table.xml`.

I updated to include those files, even we don't use them now.

`ldconfig` updates all so files. So if all libraries follow the `ld`, they should not depend on the file path.

Reply:
I also include other binaries building from dependencies. It's useful for debug.